.eslintignore

```diff
@@ -6,7 +6,3 @@ languages
 node_modules
 vendor
 legacy
-
-# ignore various storybook generated files for now (we may be able to trim this down or take ownership of these files)
-stories/0-Welcome.stories.js
-stories/1-Button.stories.js
```

Comment:
I would suggest writing stories to a subdirectory and then you can ignore the whole folder instead of having to add these one at a time.

Reply:
We won't even need to eslint ignore our stories, we'll follow our coding conventions for them too :)